### ISSUE TITLE
Support for images in MCQ options

### ISSUE THREAD

**Issue by deepansh96** [avantifellows/plio-frontend#85]
**Is your feature request related to a problem? Please describe.**
Currently, frontend supports images in MCQ questions through CMS. The support includes the styling and positioning of the image present in the question. 

In some cases, the educator/creator might also need to show images in the options. Although any images present in the options will be rendered by the frontend, the styling/positioning support is not yet available.

**Describe the solution you'd like**
The images present in the options (if any) should have a minimum size (depending on the available screen width), and should be positioned properly with appropriate CSS styles.

Basically, apply the similar styles and logic to the option images as done for the question image in this PR #62

**Comment by pritamps** [comment]
Not sure if this was clear from the problem you tried, but CMS already supports images in option in the same way as for problem.

**Comment by deepansh96** [comment]
I agree that CMS already supports images in options just like it does in question. Except the CSS, there was also a task of taking out the `<img>` tag from inside the `<p>` tag and placing it outside.

I'll rename the task -- "Apply the same set of tasks from question image to option images"
Does that sound okay?

**Comment by pritamps** [comment]
No no - no need to change anything. Just putting the info in here so that whoever picks up the task (may not be you some day :D ) has the info
